### DISCUSSION TITLE
Sending file to WhatsApp template

### DISCUSSION THREAD

**Discussion by alvestassio** [chatwoot/chatwoot#9631] [Chatwoot APIs]
**Problem description:**

I'm integrating with the api, and sending WhatsApp templates, this already works perfectly as in the example below:

```json
{
    "content": "Send template test",
    "template_params": {
        "name": "corporate_process_message_5",
        "category": "UTILITY",
        "language": "pt_BR",
        "processed_params": {
            "1": "Jose"
        }
    },
    "message_type": "outgoing"
}
```

In this example I send parameters that are necessary for the template, however I have a template that sends a pdf, but I didn't find any documentation that shows how to send a pdf to a template.

**Has anyone ever needed to upload a file to a WhatsApp template?**

**_pt_BR_**

**Descrição do problema:**

Estou fazendo a integração com a api, e enviando templates do whatsapp, isso já funciona perfeitamente como no exemplo abaixo:

```json
{
    "content": "Send template test",
    "template_params": {
        "name": "corporate_process_message_5",
        "category": "UTILITY",
        "language": "pt_BR",
        "processed_params": {
            "1": "Jose"
        }
    },
    "message_type": "outgoing"
}
```

Neste exemplo eu envio parâmetros que são necessários para o template, no entanto eu tenho um template que envia um pdf, mas nao achei nenhuma documentação que mostre como enviar um pdf para um template.

**Alguém já precisou enviar um arquivo para um template do WhatsApp?**

**Comment by raya2021** [comment]
yes needed
